Bluetooth/build.gradle

```diff
@@ -26,17 +26,17 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
Is changing the compatibility version a breaking change? (Just thinking about if the release version should be major version 2)

Reply:
@CodeReaper I did not experience any braking changes, but they still mention that there are breaking changes between the versions, so I would say let's update to version 2